hiro/windows/window.cpp

```diff
@@ -422,7 +426,48 @@ auto pWindow::_statusHeight() const -> s32 {
   }
   return height;
 }
-
+enum class Style : DWORD {
+	windowed			= WS_OVERLAPPEDWINDOW	| WS_THICKFRAME	| WS_CAPTION 	| WS_SYSMENU 	| WS_MINIMIZEBOX 	| WS_MAXIMIZEBOX,
+    aero_borderless	= WS_POPUP			| WS_THICKFRAME	| WS_CAPTION 	| WS_SYSMENU 	| WS_MAXIMIZEBOX 	| WS_MINIMIZEBOX,
+    basic_borderless	= WS_POPUP			| WS_THICKFRAME	| WS_SYSMENU 	| WS_MAXIMIZEBOX 	| WS_MINIMIZEBOX,
+	FixedStyle		= WS_SYSMENU 		| WS_CAPTION 	| WS_MINIMIZEBOX 	| WS_BORDER 		| WS_CLIPCHILDREN,
+	ResizableStyle	= WS_SYSMENU 		| WS_CAPTION 	| WS_MINIMIZEBOX 	| WS_MAXIMIZEBOX 	| WS_THICKFRAME 	| WS_CLIPCHILDREN,
+	Marty_Borderless	= WS_POPUP			| WS_VISIBLE 	| WS_CLIPSIBLINGS ,		
+};
+
+auto composition_enabled() -> bool {
```

Comment:
we don't tend to use underscores in function names, use 

`compositionEnabled`
`setShadow`, etc